sdcm/nemesis.py

```diff
@@ -241,7 +240,7 @@ def __init__(self, tester_obj, termination_event, *args, nemesis_selector=None,
         self.cluster: Union[BaseCluster, BaseScyllaCluster] = tester_obj.db_cluster
         self.loaders = tester_obj.loaders
         self.monitoring_set = tester_obj.monitors
-        nemesis_thread_name = f"Nemesis{uuid4()}"
+        nemesis_thread_name = f"Nemesis-{len(getattr(self.cluster, 'nemesis', []))}"
```

Comment:
why?

Reply:
its easier to read, instead nemesis-1235ac we'll get nemesis-0, nemesis-1 so we can easier track which nemesis thread it is

Reply:
In that case, I would rather, if the name (or just the suffix) was passed through constructor. This seems clunky, nemesis instance should not care about other nemesis instances

Reply:
ok, I'll change it back - anyway it makes not much difference